ivy/functional/frontends/numpy/logic/array_type_testing.py

```diff
@@ -69,3 +69,24 @@ def isnan(
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
+
+
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+@handle_numpy_casting_special
+def isreal(
```

Comment:
please also place the function in one file only, where `array_type_testing` is the correct one for this function